mentors/README.md

```diff
@@ -47,7 +47,6 @@ Aim for approximately 200-250 characters for the bio.
 The `link_url` and `link_text` are for linking to a personal site, if you want to.
 
 Note that if you leave a field `null`, we will use the content from your public GitHub profile.
-If you want to leave a field blank, set it to an empty string.
```

Comment:
Why do you remove this line? This seems not to be related to adding you as a mentor.

Reply:
I do think you did because you consider that line as incorrect after your former version of your bio has been denied by Travis. But instead of filing an issue as asked in the slack you simply delete that line.

I've created #265 for you. There we can talk about that issue.

You can ask with your bio until that has been decided upon and submit however it fits the new rules.

Alternatively you can get it merged in like it is now, and after that you can maybe alter the link URLs.

Either way, I still think the removal of this line is unrelated and should be reverted.